source/hardware/controllers/native_instruments_traktor_kontrol_z2.rst

```diff
@@ -0,0 +1,98 @@
+Native Instruments Traktor Kontrol Z2
+=====================================
+
+.. sectionauthor::
+   Jörg Wartenberg
+
+
+The Native Instruments Traktor Kontrol Z2 is a 2+2 channel USB :term:`HID` mixer/controller with an integrated audio interface.
+It has two full channels and two further channels with limited controls. 
+The Traktor Z2 is intended for a DVS systems and is provided with Traktor Scratch MK2 :term:`timecode` Vinyls and CDs. 
+
+-  `Manufacturer’s product page <https://www.native-instruments.com/en/products/traktor/dj-mixer/traktor-kontrol-z2/>`__
+-  `Virtual DJ Hardware Page <https://www.virtualdj.com/manuals/hardware/ni/z2.html>`__
+-  `Traktor Kontrol Z2 Manual (ZIP-file) <https://www.native-instruments.com/fileadmin/ni_media/downloads/manuals/TRAKTOR_KONTROL_Z2_Manual_All_Languages_12_2014.zip>`__
+-  `Driver/Firmware-Updates Download Page <https://www.native-instruments.com/en/support/downloads/drivers-other-files/>`__
+
+.. versionadded:: 2.4.0
+
+Drivers
+-------
+
+You can download the latest Windows & MacOS drivers and firmware from the `manufacturer’s website <https://www.native-instruments.com/en/support/downloads/drivers-other-files/>`__.
+Since the Traktor Kontrol Z2 is a USB class compliant HID and audio device, the device is plug-and-play on Linux.
+
+.. warning::
+   **NIHardwareService.exe** on Microsoft Windows
+   
+   If you've a parallel installation of the National Instruments Traktor 3 software, there will be a Windows service
+   
+   :file:`C:\\Program Files\\Common Files\\Native Instruments\\Hardware\\NIHardwareService.exe`
+   
+   installed. It starts at system boot and takes over control of the LEDs of the Traktor Kontrol Z2.
+   
+   The LEDs on the Traktor Kontrol Z2 will only reliable work with Mixxx, if you either:
+   
+   - Deinstall the Traktor 3 software
+   - Stop the Windows service NIHardwareService.exe *e.g. by Windows task manager -> Show processes from all users -> select NIHardwareService.exe -> End process*
```

Comment:
```suggestion   
   If you have a parallel installation of Traktor on Windows, there will be a :file:`NIHardwareService.exe` process that takes control of the Traktor Kontrol Z2's LEDs.
   To make the them work reliable with Mixxx, you either need to uninstall the Traktor software or stop the process by opening the task manager and killing the :file:`NIHardwareService.exe` process. 
```

Reply:
This is the most important information of the whole page, therefore it must be very visible. If this is not read, every user will be dissapointed, because it just not work.
I removed some line breaks and removed the path as suggested.

Reply:
With my suggested change, all relevant information is still there, and it's inside a warning box. What specific piece of info are you missing? 

It doesn't tell you how to use the task manager, but Windows users should know how to do that. It's not up to Mixxx to document how it works (and I think it also works differently on Windows < 10 IIRC).

Reply:
The important point is, that you don't see Windows services in the normal task manger view. I guess that an average user will not find this process without instruction. You need to elevate the user rights by "Show processes from all users" to find this service.
I'm on Windows7, therefore I'm sure the description is valid on Windows 7, as well a the latest 10 which I use on work.

Reply:
Ok, how about:

```suggestion   
   If you have a parallel installation of Traktor on Windows, there will be a :file:`NIHardwareService.exe` process that takes control of the Traktor Kontrol Z2's LEDs.
   To make the them work reliable with Mixxx, you either need to uninstall the Traktor software or stop the process by opening the task manager, selecting :guilabel:`Show processes from all users` and killing the :file:`NIHardwareService.exe` process. 
```